src/js/angular/rest/repositories.rest.service.js

```diff
@@ -36,6 +39,16 @@ function RepositoriesRestService($http) {
         });
     }
 
+    function getRepositoryModel(repoInfo) {
+        return $http.get(`${REPOSITORIES_ENDPOINT}/${repoInfo.id}`, {
+            params: {
+                location: repoInfo.location
+            }
+        }).then((response) => {
+            return repositoryConfigMapper(response.data);
```

Comment:
No. I'd rather prefer using the rest service because the repositories rest service is quite quirky to be managed. I considered creating a new service but decided to postpone this for now.